firebase/firestore/_utils.py

```diff
@@ -81,7 +87,7 @@ def _decode_datastore(value):
 	elif value.get('doubleValue'):
 		return float(value['doubleValue'])
 
-	elif value.get('stringValue'):
+	elif value.get('stringValue') is not None:
```

Comment:
Even tho mentioned line fixes the issue, won't it make more sense to use the below snippet?

```python

elif isinstance(value.get('stringValue'), str):
```